aslprep/interfaces/confounds.py

```diff
@@ -23,7 +23,6 @@
     average_cbf_by_tissue,
     compute_qei,
     dice,
-    negativevoxel,
```

Comment:
This is still used below for `percentage_negative_cbf_score` and the like.

Reply:
we can set those values from the output from compute_qei instead of calling a second function